src/screens/about/components/comparison.js

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Link } from "react-router";
+// Common
+import LOGO from "../../../../static/logotype.svg";
+import VLOGO from "victory-docs/static/logotype-hero.svg";
+
+class ComparisonChart extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>
+          When should I use FormidableCharts versus Victory?
+        </h2>
+        <p>
+          We wanted to provide developers with multiple charting options for their projects.
+        </p>
+        <p>
+          FormidableCharts is quick and easy to implement! Focus on the data with pre-built charts.
+        </p>
+        <p>
+          Victory is much more flexible, allowing users to develop pretty much any kind of chart configuration. Create fully custom charts that integrate seamlessly with the look and feel of your project.
```

Comment:
"Victory is much more flexible, allowing users to create fully custom charts that integrate seamlessly with the look and feel of a project."

I think "pretty much any kind of chart" is unnecessary.